specs/driver_spec.rb

```diff
@@ -110,27 +116,72 @@
       driver = RideShare::Driver.new(
         id: 54,
         name: "Rogers Bartell IV",
-        vin: "1C9EVBRM0YBC564DZ"
+        vin: "1C9EVBRM0YBC564DZ",
       )
       expect(driver.average_rating).must_equal 0
     end
 
     it "correctly calculates the average rating" do
       trip2 = RideShare::Trip.new(
-        id: 8,
+        id: 3,
         driver: @driver,
         passenger_id: 3,
         start_time: "2016-08-08",
         end_time: "2016-08-09",
-        rating: 1
+        rating: 1,
+      )
+
+      trip3 = RideShare::Trip.new(
+        id: 9,
+        driver: @driver,
+        passenger_id: 7,
+        start_time: Time.now.to_s,
+        end_time: nil,
+        rating: nil,
       )
+
       @driver.add_trip(trip2)
+      @driver.add_trip(trip3)
 
       expect(@driver.average_rating).must_be_close_to (5.0 + 1.0) / 2.0, 0.01
     end
   end
 
   describe "total_revenue" do
-    # You add tests for the total_revenue method
+    before do
+      @driver = RideShare::Driver.new(
+        id: 54,
+        name: "Rogers Bartell IV",
+        vin: "1C9EVBRM0YBC564DZ",
+      )
+      trip1 = RideShare::Trip.new(
+        id: 8,
+        driver: @driver,
+        passenger_id: 3,
+        start_time: "2016-08-08",
+        end_time: "2016-08-08",
+        cost: 40,
+        rating: 5,
+      )
+      trip2 = RideShare::Trip.new(
+        id: 12,
+        driver: @driver,
+        passenger_id: 7,
+        start_time: "2019-01-08",
+        end_time: nil,
+        cost: nil,
+        rating: nil
+      )
+      @driver.add_trip(trip1)
+      @driver.add_trip(trip2)
+    end
+
+    it "returns a float number" do
+      expect(@driver.total_revenue).must_be_kind_of Float
+    end
+
```

Comment:
You should also do a test when the trip is cheap enough that is less than or equal to 1.65, and when the trip is incomplete.